__tests__/transform.test.ts

```diff
@@ -528,14 +565,53 @@ it('Use Rules to get all open PRs in a Category.', async () => {
       title: '## Open PRs only',
       rules: [
         {
-          pattern: "open",
-          on_property: "status"
+          pattern: 'open',
+          on_property: 'status'
         }
       ]
     }
   ]
+  expect(buildChangelogTest(customConfig, prs)).toStrictEqual(`## Open PRs only\n\n- Still pending open pull request\n   - PR: #6\n\n`)
+})
+
+
+it('Use Rules to get current open PR and merged categorised.', async () => {
```

Comment:
@mikepenz thanks for your support!

I have the feeling I am doing something wrong.... to check if the rules works correctly, i added a test. it seems that rules are combined with `and` logic and due to that, no output is generated in this case.